src/Orc.Automation/Win32/NativeMethods.cs

```diff
@@ -0,0 +1,881 @@
+﻿// (c) Copyright Cory Plotts.
+// This source is subject to the Microsoft Public License (Ms-PL).
+// Please see http://go.microsoft.com/fwlink/?LinkID=131993 for details.
+// All other rights reserved.
+
+// ReSharper disable InconsistentNaming
+// ReSharper disable MemberCanBePrivate.Global
+// ReSharper disable UnusedMember.Local
+// ReSharper disable UnusedMember.Global
+// ReSharper disable IdentifierTypo
+#pragma warning disable CA1008
+#pragma warning disable CA1028
+#pragma warning disable CA1045
+#pragma warning disable CA1051
+#pragma warning disable CA1401
+#pragma warning disable CA1806
+#pragma warning disable CA1815
+#pragma warning disable CA1819
+#pragma warning disable CA2101
+
+namespace Orc.Automation;
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+using System.Xml.Serialization;
+using Microsoft.Win32.SafeHandles;
+
+public static class NativeMethods1
```

Comment:
This class name is invalid.

Also:

1. Split up into structs, methods, etc. See Orchestra for a good use case